src/main/deploy/shooterData.csv

```diff
@@ -1,20 +1,16 @@
 
-Distance (inches),Angle (degrees),Velocity (RPM)
+Distance (inches),Angle (degrees),Velocity (RPM), Time of Flight (seconds)
 Actual point(s)
 //0,   5.05,  3092
 //34,  5.05,  3200
 
-80,  15, 2200 # Working
-100, 21,  2200 
-120, 22.18, 2300
-140, 23.3,  2400
-162, 25, 2500 
-180, 27, 2700 # Close launchpad 
-200, 29, 2866
-220, 29.5, 2950
-240, 29.6, 3070
-260, 29.42, 3126
-
-//230, 27.30, 3250 # Far launchpad
-
-//260, 31,    3522 
+80,  15,    2200   0    # Working
```

Comment:
Note that CSV is short for "Comma Separated Values"- The TOF entries are missing a comma before them, so no line is valid and the interpolating tree map is empty. (Side note: There's CSV debug info that appears between "Robot program starting" and "Robot program startup complete".)